src/commands/build/internal/getLaunchPageCode.ts

```diff
@@ -1,18 +1,18 @@
 import { ServerType } from '@sasjs/utils/types'
 
-const SAS9Code = (streamServiceName: string) => `
+const SAS9Code = (streamServiceName: string, appLoc: string) => `
 options notes;
 data _null_;
  format url $256.; 
  rc=METADATA_GETURI("Stored Process Web App",url);
  url=coalescec(url,"localhost/SASStoredProcess");
  putlog "NOTE: SASjs Streaming App Created! Check it out here:" ;
  putlog "NOTE- ";putlog "NOTE- ";putlog "NOTE- ";putlog "NOTE- ";
- putlog "NOTE- " url +(-1) "?_program=&appLoc/services/${streamServiceName}" ;
+ putlog "NOTE- " url +(-1) "?_program=${appLoc}/services/${streamServiceName}" ;
```

Comment:
This should remain as `&appLoc` so that the generated build.sas file can be modified by the end user by simply updating the `%let apploc=xxx;` at the start (otherwise it will get hard coded from the sasjsconfig attribute)

Reply:
fixed